examples/intro-to-r.md

```diff
@@ -0,0 +1,250 @@
+[title]: - "Run R scripts on OSG"
+[TOC]
+
+## Overview
+This tutorial describes how to run R scripts on the OSG. We'll first run the program locally as a test.  After that we'll create a submit file, submit it to OSG using OSG Connect, and look at the results when the jobs finish. Finally, we will talk about how to use custom R libraries on OSG Connect.
+
+## Run R scripts on OSG
+### Access R on the submit host
+
+First we'll need to create a working directory, you can either run `$ tutorial R` or type the following:
+
+	$ mkdir tutorial-R; cd tutorial-R
+
+R is installed using modules on OSG. To load this modules and access R, enter:
+
+	$ module load r/3.5.1-py2.7
+	
+
+Now, we can try to run R:
+
+	$ R
+	
+	R version 3.5.1 (2018-07-02) -- "Feather Spray"
+	Copyright (C) 2018 The R Foundation for Statistical Computing
+	Platform: x86_64-pc-linux-gnu (64-bit)
+
+	R is free software and comes with ABSOLUTELY NO WARRANTY.
+	You are welcome to redistribute it under certain conditions.
+	Type 'license()' or 'licence()' for distribution details.
+
+	  Natural language support but running in an English locale
+
+	R is a collaborative project with many contributors.
+	Type 'contributors()' for more information and
+	'citation()' on how to cite R or R packages in publications.
+
+	Type 'demo()' for some demos, 'help()' for on-line help, or
+	'help.start()' for an HTML browser interface to help.
+	Type 'q()' to quit R.
+
+	> 
+
+Great! R works. You can quit out with `q()`. 
+
+	> q()
+	Save workspace image? [y/n/c]: n
+	$
+
+### Run R code
+
+Now that we can run R, let's create a small script. Create the file `hello_world.R` that contains the following:
+
+	print("Hello World!")
+
+R normally runs as an interactive shell, but it is easy to run in batch mode too.
+
+	$ $ Rscript --no-save hello_world.R
+	[1] "Hello World!"
+
+
+### Build the HTCondor job
+
+To prepare msng R job to run on OSG, we need to create a wrapper for our R environment, based on the setup we did in previous sections. Create the file `R-wrapper.sh`:
+
+	#!/bin/bash
+	 
+	EXPECTED_ARGS=1
+	 
+	if [ $# -lt $EXPECTED_ARGS ]; then
+	  echo "Usage: R-wrapper.sh file.R"
+	  exit 1
+	else
+	  module load r/3.5.1-py2.7
+	  Rscript $1
+	fi
+
+Notice here that we're using Rscript (equivalent to `R CMD BATCH`). It accepts the script as command line argument, it makes `R` much less verbose, and it's easier to parse the output later. If you run it at the command line, you should get similar output as above. This lets the wrapper launch `R` on any generic worker node under HTCondor.
```

Comment:
I was torn on this. I did this because the existing R docs used arguments to pass the script name and I thought keeping the r-wrapper script "generic" would be helpful for users who might want to reuse them for other jobs.

A discussion on this would be good.

Reply:
I have mixed feelings about the "magic" of arguments, but I think it makes it a *lot* easier to reuse the wrapper script -- people don't have to create a bunch of different shell scripts for different R scripts.  Basically what you said.  :)

maybe we can discuss next week.

Reply:
Removed all arguments from the wrapper script and removed the possibly confusing `if` statement.